traffic/data/adsb/opensky_impala.py

```diff
@@ -875,7 +866,7 @@ def history(
 
         cumul = []
         sequence = list(split_times(start, stop, date_delta))
-        columns = ", ".join(self._impala_columns)
```

Comment:
This seemed completely redundant given that later in the original code it was completelt overwritten.

Reply:
It's only overwritten if `count_airports_params > 0` right?

Reply:
The
```python
", ".join(f"sv.{field}" for field in self._impala_columns)
```
was repeated (overwriting whatever was done before, without the `sv.` prefix) if `count_airports_params > 0`.
But in the new code the  `sv.` prefix is always there and so I initialized `columns` with the prefix
and added the `est.` columns for the `count_airports_params > 0` case.